src/v2/Components/NavBar/constants.ts

```diff
@@ -1,9 +1,7 @@
-import { space } from "@artsy/palette"
-
 export const NAV_BAR_BORDER_OFFSET = 1
-export const NAV_BAR_TOP_TIER_HEIGHT = space(6) - NAV_BAR_BORDER_OFFSET
+export const NAV_BAR_TOP_TIER_HEIGHT = 60 - NAV_BAR_BORDER_OFFSET
 export const NAV_BAR_BOTTOM_TIER_HEIGHT = 44
-export const MOBILE_BANNER_HEIGHT = 64
+export const MOBILE_BANNER_HEIGHT = 62
```

Comment:
This value will change when we switch it to V3 — V3 buttons are `50px` high, V2 buttons are `41px`